### PR TITLE
 Add doc folder initialisation

### PR DESCRIPTION
**Pull request by mcrinquand**
prestaconcept/open-source-management#55 

Add installation documentation file in skeleton

Add installation doc for initializer-bundle

### CODE REVIEW

File: Resources/doc/001-installation.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            ...
+            "Presta\\InitializerBundle\\Composer\\ScriptHandler::install",
+        ],
+    }
+
+### You have to activate the bundle on AppKernel.php
+
+    new Presta\InitializerBundle\PrestaInitializerBundle(),
+
+### Supprimer le README
```
Comment (nicolas-bastien) [inline Resources/doc/001-installation.md]: french ?

File: README.md [hunk 2]
```diff
@@ -25,6 +25,12 @@ to centralize our open source activity.
 
 :speech_balloon: If you need help about this project you can [post a message on our google group][3]
 
+## Documentation ##
+
+Vous trouverez la documentation projet dans le répertoire /Resources/doc :
```
Comment (nicolas-bastien) [inline README.md]: french ?

File: Resources/skeleton/doc/001-installation.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        app_path + "/cache",
+        app_path + "/logs",
+        web_path + "/uploads"
+    ]
+
+    set :shared_children,       [
+        app_path + "/logs",
+        app_path + "/exports",
+        web_path + "/uploads"
+    ]
```
Comment (nicolas-bastien) [inline Resources/skeleton/doc/001-installation.md]: Détails des répertoires de l'application 

- /web/upload/wysiwyg : .....

File: Resources/skeleton/doc/001-installation.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+* test-clean : réinitialisation de la base de donnée de test
+* ai : installation des assets en mode par défaut (hard copy)
+* ais :  installation des assets en mode symlink
+* cs : Exécute le checkstyle PSR2
+* cs-fixer : correction automatique du checkstyle
+* cc : vidage du cache
+* pdc : Exécute PHPDocBlockChecker
+   
+## Installation en production
+
```
Comment (nicolas-bastien) [inline Resources/skeleton/doc/001-installation.md]: ### configuration

**Comment by mcrinquand** [comment]
@nicolas-bastien it's ok

**Pull request merged**